src/App.js

```diff
@@ -289,4 +291,8 @@ const ProtestListWrapper = styled.div`
   }
 `;
 
+const ProtestListHead = styled.div`
+  margin-bottom: 0.5rem;
```

Comment:
Better to use 8px - https://stackoverflow.com/a/11803273/3232187
(also in order to be consistent with the project)